scrapers/CopyFromScene.py

```diff
@@ -3,83 +3,24 @@
 import requests
 import sys
 
-
-url="http://localhost:9999/graphql"
-headers = {
-    "Accept-Encoding": "gzip, deflate, br",
-    "Content-Type": "application/json",
-    "Accept": "application/json",
-    "Connection": "keep-alive",
-    "DNT": "1"
-}
-
-def __callGraphQL(query, variables=None):
-
-    json = {}
-    json['query'] = query
-    if variables != None:
-        json['variables'] = variables
-
-    # handle cookies
-    response = requests.post(url, json=json, headers=headers)
-
-    if response.status_code == 200:
-        result = response.json()
-        if result.get("error", None):
-            for error in result["error"]["errors"]:
-                raise Exception("GraphQL error: {}".format(error))
-        if result.get("data", None):
-            return result.get("data")
-    else:
-        raise Exception(
-            "GraphQL query failed:{} - {}. Query: {}. Variables: {}".format(response.status_code, response.content, query, variables))
-
-def findGallery(id):
-    query = """query findGallery($gallery_id: ID!){
-findGallery(id: $gallery_id){
-id
-path
-scenes{
-title
-url
-date
-rating
-details
-organized
-studio{
-  id
-  name
-}
-performers{
-id
-  name
-}
-tags{
-id
-  name
-}
-}
-}
-}"""
-
-    variables = {'gallery_id': id, }
-    result = __callGraphQL(query, variables)
-    if result is not None:
-        return result["findGallery"]
-    return None
-
+try:
+    import py_common.graphql as graphql
+    import py_common.log as log
+except ModuleNotFoundError:
+    print("You need to download the folder 'py_common' from the community repo! (CommunityScrapers/tree/master/scrapers/py_common)", file=sys.stderr)
+    sys.exit()
 
 if sys.argv[1] == "gallery_query":
     fragment = json.loads(sys.stdin.read())
-    print("input: " + json.dumps(fragment),file=sys.stderr)
-    result = findGallery(fragment['id'])
+    log.debug("input: " + json.dumps(fragment),file=sys.stderr)
+    result=graphql.getGallery(fragment['id'])
     if not result:
-        print(f"Could not determine details for gallery: `{fragment['id']}`",file=sys.stderr)
+        log.info(f"Could not determine details for gallery: `{fragment['id']}`",file=sys.stderr)
         print("{}")
     else:
         if len(result["scenes"])  > 0:
-            print (json.dumps(result["scenes"][0]))
+            s=result["scenes"][0]
+            res={"title":s["title"],"details":s["details"],"url":s["url"],"date":s["date"],"rating":s["rating"],"studio":s["studio"],"performers":s["performers"],"tags":s["tags"]}
```

Comment:
- scraped results dont have `rating` 

- if we want to `clean` the results propely then we we need a separate function as
for studio,tags,performers we need to only keep the `name` and not the `id`